mender/templates/gui-deploy.yaml

```diff
@@ -42,22 +42,28 @@ spec:
       {{- with .Values.gui.affinity }}
       affinity: {{ tpl (toYaml .) $ | nindent 8 }}
       {{- end }}
+{{- if .Values.gui.podSecurityContext.enabled }}
+      securityContext: {{- omit .Values.gui.podSecurityContext "enabled" | toYaml | nindent 8 }}
+{{- end }}
 
       containers:
       - name: gui
         image: {{ .Values.gui.image.registry }}/{{ .Values.gui.image.repository }}:{{ .Values.gui.image.tag }}
         imagePullPolicy: {{ .Values.gui.image.imagePullPolicy }}
+{{- if .Values.gui.containerSecurityContext.enabled }}
+        securityContext: {{- omit .Values.gui.containerSecurityContext "enabled" | toYaml | nindent 10 }}
+{{- end }}
         resources:
 {{ toYaml .Values.gui.resources | indent 10 }}
 
         # Readiness/liveness probes
         livenessProbe:
           tcpSocket:
-            port: 80
+            port: {{ .Values.gui.httpPort }}
           periodSeconds: 5
         readinessProbe:
           tcpSocket:
-            port: 80
+            port: {{ .Values.gui.httpPort }}
```

Comment:
Unfortunately the `gui` container is not currently supporting non-root users. Please remove the modifications here for now.